.gitmodules

```diff
@@ -14,16 +14,9 @@
 	path = contracts/libc++/upstream
 	url = https://github.com/EOSIO/libcxx.git
 	branch = eosio
-[submodule "externals/binaryen"]
```

Comment:
`contracts/musl/upstream` and `contracts/libc++/upstream` above can go as well

Reply:
that's interesting `git submodule` didn't show those

Reply:
Maybe because parent directory does not exist, but who knows.